lib/dockerregistry/inventory_test.go

```diff
@@ -334,6 +336,342 @@ could not find source registry in '[gcr.io/bar/banana gcr.io/cat/subdir/A/banana
 	}
 }
 
+func TestParseManifestsFromDir(t *testing.T) {
+	// TEST_SRCDIR is the test directory that Bazel creates during test runs.
+	pwd := filepath.Join(
+		os.Getenv("TEST_SRCDIR"),
+		"__main__/lib/dockerregistry/inventory_test/TestParseManifestsFromDir")
```

Comment:
I believe the reliable way to do this is to create a directory `lib/dockerregistry/tests`; gazelle should automatically include it, and then `cwd` during test execution will be `lib/dockerregistry`, so you should be able to use `tests/TestParseManifestsFromDir`...  I think!

Reply:
Noted... I will come back to this when bazel updates and this test breaks. :upside_down_face: